File: guilherme/Problema0008/Problema0008.py
deposito = float(input("Digite um valor de depósito: "))
txJuros = float(input("Digite o valor da taxa de juros(%): "))
rendimento = deposito*(txJuros/100)
valorTotal = rendimento+deposito
print("O rendimento foi de: R${}".format (rendimento))
print("O valor total depois do rendimento é: R${}".format (valorTotal))

File: guilherme/Problema0008/Problema0008.java
import java.util.Scanner;

public class Problema0008 {
    static Scanner input = new Scanner(System.in);
   
    public static void main(String[] args){
        //Entrada
        float deposito, txJuros, rendimento, valorTotal;
        System.out.println("Digite um valor de depósito: ");
        deposito = Float.valueOf(input.nextLine());
        System.out.println("Digite o valor da taxa de juros(%): ");
        txJuros = Float.valueOf(input.nextLine());
        
        //Processamento
        rendimento = deposito*(txJuros/100);
        valorTotal = deposito+rendimento;

        //Saída
        System.out.println("O rendimento foi de: R$"+rendimento);
        System.out.println("O valor total depois do rendimento é: R$"+valorTotal);
    }
}